debezium-schema-generator/pom.xml

```diff
@@ -18,6 +18,10 @@
       <groupId>io.debezium</groupId>
       <artifactId>debezium-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.debezium</groupId>
+      <artifactId>debezium-storage-kafka</artifactId>
```

Comment:
Should we add a comment here that this is a temporary fix?

Reply:
I don't think so, but we do need to log a Jira issue tho as a follow-up task to think about the architecture used and to see whether we can find a better, more robust way to deal with the database history configuration.  I know that @blcksrx is looking into the validation concerns, so we just don't want to lose sight of it.

Reply:
https://issues.redhat.com/browse/DBZ-5564